lib/payonline/payment_gateway.rb

```diff
@@ -46,13 +16,29 @@ def filter_params
     }
 
     def initialize(params = {})
-      @params = Params.new(params.with_indifferent_access)
+      @params = prepare_params(params)
     end
 
     def payment_url(type: :card, language: :ru)
       params = Payonline::Signature.sign_params(@params, SIGNED_PARAMS)
 
       "#{BASE_URL}/#{language}/payment/#{PAYMENT_TYPE_URL[type]}?#{params.to_query}"
     end
+
+    private
+
+    def prepare_params(params)
+      params.with_indifferent_access
+        .slice(*REQUIRED_PARAMS)
```

Comment:
@yuri-zubov If you paste this code into terminal it will work, because there's a method call after the first dot. Anyway, should we optimize for terminals or for readability? I think readability should be the first priority and this code should not be used in terminal. [Style guide regarding multi-line method calls](https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains).